### ISSUE TITLE
Running time for tests?

### ISSUE THREAD

**Issue by DerekCresswell** [karansher/computer-graphics-ray-casting#32]
I saw the comment at the bottom of the readme about release mode. Running the bunny without release takes... well I got bored of waiting!

On release mode, it takes maybe <30s. I am using one of the linear equation solvers in Eigen. I will try switching solving methods to get something faster, but I am wondering if these times will work for whatever testing is run on them!

**Comment by panuelosj** [comment]
Yes, we'll be running in release mode so as long as it runs in reasonable time there it should be okay.
